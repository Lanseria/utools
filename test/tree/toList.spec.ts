import { treeToList } from "@lanseria/utools";
import { data2 } from "../mock";

describe("Tree To List 树转为数组操作", () => {
  it("普通测试", () => {
    const list = treeToList(data2);
    expect(list).toMatchSnapshot();
  });
});
