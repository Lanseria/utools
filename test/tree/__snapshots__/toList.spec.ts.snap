// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tree To List 树转为数组操作 普通测试 1`] = `
Array [
  Object {
    "name": "a",
  },
  Object {
    "name": "a1",
  },
  Object {
    "name": "a11",
  },
  Object {
    "name": "a12",
  },
  Object {
    "name": "a2",
  },
  Object {
    "name": "a21",
  },
  Object {
    "name": "a22",
  },
  Object {
    "name": "a3",
  },
  Object {
    "name": "a31",
  },
  Object {
    "name": "a32",
  },
]
`;
